packages/components/Button/src/Button.types.ts

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { ViewProps, ViewStyle, ColorValue } from 'react-native';
+import { ViewProps, ViewStyle, ColorValue, GestureResponderEvent, AccessibilityActionEvent } from 'react-native';
 import { TextProps } from '@fluentui-react-native/experimental-text';
 import { FontTokens, IBorderTokens, IColorTokens, IShadowTokens, LayoutTokens } from '@fluentui-react-native/tokens';
-import { IFocusable, IPressableHooks, IWithPressableOptions } from '@fluentui-react-native/interactive-hooks';
+import { IFocusable, IPressableHooks, IWithPressableOptions, KeyPressEvent } from '@fluentui-react-native/interactive-hooks';
```

Comment:
It's probably good feedback to RN-macOS / windows that there should be a common "KeyPresEvent" type so that we can import it like the others from line 2